chassis/core/src/main/java/com/griddynamics/jagger/user/test/configurations/JTestGroup.java

```diff
@@ -1,51 +1,36 @@
 package com.griddynamics.jagger.user.test.configurations;
 
+import java.util.Collections;
 import java.util.List;
 
 /**
  * Describes the group of {@link JTest} that should run in parallel.
  */
 public class JTestGroup {
-    private String id;
-    private List<JTest> tests;
+    private final String id;
+    private final List<JTest> tests;
 
-
-    public static Builder builder() {
-        return new Builder();
+    public static Builder builder(Id id, List<JTest> tests) {
```

Comment:
may be we can merge next builder with this
 builder(Id id, JTest test, JTest... tests) ?